corehq/apps/app_manager/analytics.py

```diff
@@ -3,15 +3,15 @@
 
 def update_analytics_indexes():
     from .models import Application
-    Application.get_db().view('exports_forms/by_xmlns', limit=1).all()
+    Application.get_db().view('exports_forms_by_app/view', limit=1).all()
 
 
 def get_exports_by_application(domain):
     from .models import Application
     return Application.get_db().view(
-        'exports_forms/by_xmlns',
-        startkey=['^Application', domain],
-        endkey=['^Application', domain, {}],
+        'exports_forms_by_app/view',
+        startkey=[domain, {}],
+        endkey=[domain, {}, {}],
```

Comment:
This isn't quite equivalent - it will now also match user registration applications.  Think that's something we need to worry about?